test/model/content_test.dart

```diff
@@ -421,14 +421,89 @@ void main() {
       '<br>\n</p>\n</blockquote>',
     [const QuotationNode([MathBlockNode(texSource: r'\lambda')])]);
 
-  testParse('parse image',
-    // "https://chat.zulip.org/user_avatars/2/realm/icon.png?version=3"
-    '<div class="message_inline_image">'
+  group('Parsing images', () {
+    testParse('single image',
+      // "https://chat.zulip.org/user_avatars/2/realm/icon.png?version=3"
+      '<div class="message_inline_image">'
         '<a href="https://chat.zulip.org/user_avatars/2/realm/icon.png?version=3">'
-        '<img src="https://chat.zulip.org/user_avatars/2/realm/icon.png?version=3">'
-        '</a></div>', const [
-      ImageNode(srcUrl: 'https://chat.zulip.org/user_avatars/2/realm/icon.png?version=3'),
-    ]);
+          '<img src="https://chat.zulip.org/user_avatars/2/realm/icon.png?version=3"></a></div>',
+      const [
+        ImageNodes([
+          ImageNode(srcUrl: 'https://chat.zulip.org/user_avatars/2/realm/icon.png?version=3'),
+        ]),
+      ]);
+
+    testParse('parse multiple images',
+      // "https://chat.zulip.org/user_avatars/2/realm/icon.png?version=3\nhttps://chat.zulip.org/user_avatars/2/realm/icon.png?version=4"
+      '<p>'
+        '<a href="https://chat.zulip.org/user_avatars/2/realm/icon.png?version=3">https://chat.zulip.org/user_avatars/2/realm/icon.png?version=3</a><br>\n'
+        '<a href="https://chat.zulip.org/user_avatars/2/realm/icon.png?version=4">https://chat.zulip.org/user_avatars/2/realm/icon.png?version=4</a></p>\n'
+        '<div class="message_inline_image">'
+          '<a href="https://chat.zulip.org/user_avatars/2/realm/icon.png?version=3">'
+            '<img src="https://uploads.zulipusercontent.net/f535ba07f95b99a83aa48e44fd62bbb6c6cf6615/68747470733a2f2f636861742e7a756c69702e6f72672f757365725f617661746172732f322f7265616c6d2f69636f6e2e706e673f76657273696f6e3d33"></a></div>'
+        '<div class="message_inline_image">'
+          '<a href="https://chat.zulip.org/user_avatars/2/realm/icon.png?version=4">'
+            '<img src="https://uploads.zulipusercontent.net/8f63bc2632a0e41be3f457d86c077e61b4a03e7e/68747470733a2f2f636861742e7a756c69702e6f72672f757365725f617661746172732f322f7265616c6d2f69636f6e2e706e673f76657273696f6e3d34"></a></div>',
```

Comment:
```suggestion
      '<p>'
        '<a href="https://chat.zulip.org/user_avatars/2/realm/icon.png?version=3">https://chat.zulip.org/user_avatars/2/realm/icon.png?version=3</a><br>\n'
        '<a href="https://chat.zulip.org/user_avatars/2/realm/icon.png?version=4">https://chat.zulip.org/user_avatars/2/realm/icon.png?version=4</a></p>\n'
      '<div class="message_inline_image">'
        '<a href="https://chat.zulip.org/user_avatars/2/realm/icon.png?version=3">'
          '<img src="https://uploads.zulipusercontent.net/f535ba07f95b99a83aa48e44fd62bbb6c6cf6615/68747470733a2f2f636861742e7a756c69702e6f72672f757365725f617661746172732f322f7265616c6d2f69636f6e2e706e673f76657273696f6e3d33"></a></div>'
      '<div class="message_inline_image">'
        '<a href="https://chat.zulip.org/user_avatars/2/realm/icon.png?version=4">'
          '<img src="https://uploads.zulipusercontent.net/8f63bc2632a0e41be3f457d86c077e61b4a03e7e/68747470733a2f2f636861742e7a756c69702e6f72672f757365725f617661746172732f322f7265616c6d2f69636f6e2e706e673f76657273696f6e3d34"></a></div>',
```

The `div` elements aren't nested in the `p`, so the indentation should reflect that.